packages/carrot-scripts/src/scripts/start/start-ipfs-node.ts

```diff
@@ -0,0 +1,98 @@
+import ora from "ora";
+import { join, resolve } from "path";
+import { readFileSync, writeFileSync, mkdirSync, rmSync, existsSync } from "fs";
+import { homedir } from "os";
+import { $ } from "execa";
+import { createIPFSDaemon } from "../../ipfs-daemon/createIPFSDaemon.js";
+import { create as createIPFSClient } from "ipfs-http-client";
+
+const specificationLocation = resolve("./packages/frontend/src/base.json");
+
+const GATEWAY_PORT = 9090;
+const HTTP_API_PORT = 5002;
+const RPC_API_PORT = 5003;
+const REPO_PATH = join(homedir(), ".cct/ipfs");
+
+export interface StartIPFSNodeReturnValue {
+    specificationCid: string;
+    gatewayPort: number;
+    httpAPIPort: number;
+    rpcAPIPort: number;
+}
+
+export const startIPFSNode = async (): Promise<StartIPFSNodeReturnValue> => {
+    const ipfsNodeSpinner = ora();
+    ipfsNodeSpinner.start("Starting up local IPFS node");
+    let specificationCid;
+    try {
+        const configPath = join(REPO_PATH, "./config");
+        if (existsSync(REPO_PATH))
+            rmSync(REPO_PATH, {
+                force: true,
+                recursive: true,
+            });
+
+        mkdirSync(REPO_PATH, {
+            recursive: true,
+        });
+        await $({ env: { IPFS_PATH: REPO_PATH } })`ipfs init`;
+        const jsonConfig = readFileSync(configPath, "utf8");
+        const config = JSON.parse(jsonConfig);
+        config.Addresses.API = `/ip4/127.0.0.1/tcp/${HTTP_API_PORT}`;
+        config.Addresses.Gateway = `/ip4/127.0.0.1/tcp/${GATEWAY_PORT}`;
+        config.API.HTTPHeaders = { "Access-Control-Allow-Origin": ["*"] };
+        config.Peering = {
+            Peers: [
```

Comment:
What are these peers?

Reply:
These are our IPFS nodes where Carrot data is pinned through our pinner.